README.rst

```diff
@@ -0,0 +1,87 @@
+=====================
+TRAINING COMMON STUFF
+=====================
+
+This repository contains the common stuff to build presentations with pandoc
```

Comment:
Can you replace "stuff" with boilerplate or similar, "stuff" just sound awful. ;)